recipe/meta.yaml

```
@@ -1,37 +1,44 @@
-{% set version = "4.5.1" %}
+{% set version = "4.10" %}
 
 package:
   name: libpysal
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/l/libpysal/libpysal-{{ version }}.tar.gz
-  sha256: 60bbe3291aceccdf29c8ad9cf2eec76633f67ab7756d0353853f7f34a756ae12
-
+  sha256: 4d3127802c7c06289ffb211d10202dd8d14b10039a81249920769ce4b987e3b5
+  patches:
+    - patches/0001-test_weights_relative_to_absolute.patch
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: True  # [py<310 or s390x]
 
 requirements:
+  build:
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
-    - python >=3.7
+    - python
     - pip
-    - pytest-runner
     - setuptools
```

```suggestion
    - setuptools
    - setuptools_scm
```
https://github.com/pysal/libpysal/blob/v4.10/pyproject.toml#L2C34-L2C48